cmd/flightctl/main.go

```diff
@@ -238,7 +237,7 @@ func getClient() (*client.ClientWithResponses, error) {
 	return client.NewClientWithResponses(serverUrl, client.WithHTTPClient(httpClient))
 }
 
-func RunGet(kind, name string, labelSelector, fleetName *string, output string, limit *int32, cont *string) error {
+func RunGet(kind, name string, labelSelector, fleetName *string, output string, limit *int32, cont *string) error { //nolint:gocyclo
```

Comment:
gocyclo should probably all be fixed  vs nolint?

Reply:
It's a big refactor, I'll open a separate ticket

Reply:
I guess an issue to track would work if the work is complex.

Reply:
Opened [MGMT-16746](https://issues.redhat.com/browse/MGMT-16746)